src/_components/Category/CategoryContainer.jsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import CategoryComponent from '../../components/Category/AddCategoryComponent';
+import { createCategory } from '../../_actions/category.actions';
+import { ToastMessage } from '../../_utils/ToastMessage';
+import resetToastMessageContent from '../../_actions/resetToastMessage.actions';
+
+class CategoryContainer extends React.Component {
+      state = {
```

Comment:
The indentation is off. Please use `2 spaces`

Reply:
fixed.